composer.json

```diff
@@ -22,6 +22,7 @@
 		"automattic/jetpack-error": "@dev",
 		"automattic/jetpack-jitm": "@dev",
 		"automattic/jetpack-logo": "@dev",
+		"automattic/jetpack-mobile": "@dev",
```

Comment:
calling it jetpack-mobile doesn't completely make sense, since it is also used to detect desktop devices, and potentially other devices. So you might call it `automattic/jetpack-device-detection` or `automattic/jetpack-devices`